jQuery/styles.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  margin: 20px 5%;
+}
+
+#hh-menu {
+  border-right: solid 1px black;
+  position: fixed;
+  width: 240px;
+  height: 100%;
+}
+
+#hh-menu > select {
```

Comment:
remember up there when I put a class of `menu-label` on a label? you could do the same with `select`

```
.menu-option {
  width: 80%;
  margin-left: 15%;
 margin-right: 5%;
}
```

What you're doing with these margins is... eugh. 
but we'll get to that in another pass :)